apps/etl/extraction/sources/base/extract.py

```diff
@@ -32,7 +32,7 @@ def pull_data(self, source: int, retry_count: int, timeout: int = 30, ext_object
                 instance_obj.attempt_no = retry_count
                 instance_obj.save(update_fields=["resp_code", "attempt_no"])
             except ExtractionData.DoesNotExist:
-                raise ObjectDoesNotExist("ExtractionData object with ID {ext_object_id} not found")
+                raise ObjectDoesNotExist(f"ExtractionData object with ID {ext_object_id} not found")
```

Comment:
Let's remove this except

Also, make sure to change the another one as well `requests.exceptions.RequestException`